### PR TITLE
Delete samples/gui/front-end directory.

### PR DESCRIPTION
**Pull request by Mohammad-nassar10**
The GUI frontend is not part of the core fybrik, and is moved into a separate repository [fybrik-front-end](https://github.com/fybrik/fybrik-front-end). We keep the `rest-server` directory in `fybrik/samples` that replaces the `gui` directory. 

Signed-off-by: nassarmu10 <mohammad.nassar@ibm.com>

### CODE REVIEW

**Comment by roytman** [comment]
We have to update the `samples/gui` directory too, some files ( or part of files) should be moved to a new repository, the directory itself should be renamed to something like restServer.
Let's review it with @shlomitk1 during today's call.

**Comment by shlomitk1** [comment]
Why do we still have double hierarchy: rest-server/server instead of just rest-server?

**Review by roytman — COMMENTED** [review]
- I don't think we need gui_configmap.yaml
- all make commands in the README file should be run from server dir.
- you have to update the deploy.sh - change deployment name from `gui` to `rest-server`
- tried to run the deploy.sh (after building the docker image), the pod was not able to start, the image name was wrong. it requires definition of DOCKER_HOSTNAME and DOCKER_NAMESPACE
- all lines from 74 in README file should be removed.

**Comment by roytman** [comment]
Actually, you don't need the sub-directory `server`, pls just move all files into the `rest-server` dir

**Review by roytman — COMMENTED** [review]
did you try to run the rest-server ?

File: samples/rest-server/Makefile [hunk 1]
```diff
@@ -33,4 +33,11 @@ docker-build:
 	docker build . -t ${IMG} -f Dockerfile
```
[the remark this answers is not in the document]
Comment (Mohammad-nassar10) [reply samples/rest-server/Makefile:33]: The `Dockerfile` should be outside the `datauser` directory.

[the remark this answers is not in the document]
Comment (roytman) [reply samples/rest-server/Makefile:33]: but it is there

[the remark this answers is not in the document]
Comment (Mohammad-nassar10) [reply samples/rest-server/Makefile:33]: Right, I will move it outside `datauser`.

**Review by roytman — APPROVED** [review]

**Pull request merged**